src/multiserialviewer/serial_data/serialDataReceiver.py

```diff
@@ -13,7 +14,6 @@ def __init__(self, settings: SerialConnectionSettings):
         self.__serialPort.setStopBits(settings.stopbits)
         self.__serialPort.setDataBits(settings.dataBits)
         self.__settings = settings
```

Comment:
the settings timeout value is not used/needed any more. Please remove it

Reply:
Do we have to make a v3 settings tag then ?

Reply:
No, I killed compatibility completely in my settings rework branch. So it would not help here and besides that, no one is using it anyways... ;)